samples/CSharp/SourceGenerators/GeneratedDemo/GeneratedDemo.csproj

```diff
@@ -7,16 +7,23 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <AdditionalFiles Include="MainSettings.xmlsettings" />
-    <None Include="MainSettings.xmlsettings">
-      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
-    </None>
+    <AdditionalFiles Include="People.csv" CsvLoadType="Startup" />
+    <AdditionalFiles Include="Cars.csv" CsvLoadType="OnDemand" CacheObjects="true" />
+
+    <AdditionalFiles Include="MainSettings.xmlsettings" CopyToOutputDirectory="Always" />
+    <None Include="MainSettings.xmlsettings" CopyToOutputDirectory="Always" /> <!-- TODO: remove this when AdditionalFiles supports CopyToOutputDirectory -->
```

Comment:
❗ _Never_ use `Always` for this property (it disables incremental builds and all up-to-date checks for the project). Use `PreserveNewest` instead.